packages/devtools_app/lib/src/screens/vm_developer/object_inspector_view.dart

```diff
@@ -66,13 +66,13 @@ class _ObjectInspectorViewState extends State<_ObjectInspectorView> {
     final objRef = node.object;
     final location = node.location;
 
+    if (location != null) {
```

Comment:
Any reason this was moved here?

Reply:
Yes, the current location needed to be updated with the selected script node before pushing the object to the viewport, in order to have the correct script associated with that object.